pkg/types/cluster.go

```diff
@@ -28,6 +28,14 @@ type Cluster struct {
 	CreatedAt time.Time     `json:"created_at"`
 	ExpiresAt time.Time     `json:"expires_at"`
 
+	TTL string `json:"ttl"`
+
+	CreditsPerHourPerCluster int64 `json:"credits_per_hour_per_cluster"`
+	FlatFee                  int64 `json:"flat_fee"`
+	TotalCredits             int64 `json:"total_credits"`
+
+	EstimatedCost int64
```

Comment:
Not sure what you are going for here. Shouldn't this have some sort of json tag? If you were to list clusters --output json this probable doesn't look right the way it is.

Reply:
You're right. I was thinking as `EstimatedCost` is calculated (part of dry-run, or calculated for cluster list), it didn't have to go into json, but totally forgot about `--output`